index.js

```diff
@@ -271,7 +287,43 @@ function logError(context, error){
             return null;
         }
       } while (nbResults === pageSize);
-  
+
+    let hSeverity = "";
+    if (version >= "8.1" && !data.noSecurityHotspot) {
+      page = 1;
+      do {
+        try {
+            const response = await got(`${sonarBaseURL}/api/hotspots/search?projectKey=${sonarComponent}&ps=${pageSize}&p=${page}&statuses=${HOTSPOT_STATUSES}`, {
+                agent,
+                headers
+            });
+            page++;
+            const json = JSON.parse(response.body);
+            nbResults = json.hotspots.length;
+            data.issues = data.issues.concat(json.hotspots.map(hotspot => {
+              hSeverity = hotspotSeverities[hotspot.vulnerabilityProbability];
+              if (hSeverity === undefined) {
+                hSeverity = "INFO";
```

Comment:
almost forgot: small change here  would be nice too: in case of mapping error, we shouldn't hide that behind an "INFO" level that nobody would look at; how about the average level instead: MAJOR

plus logging of error would be nice in that case. something like `console.error("Unknown hotspot severity: %s", hotspot.vulnerabilityProbability);`